### PR TITLE
Feature surrogate

### PR DESCRIPTION
**Pull request by planes**
This PR makes it possible to run the VLM subsonic only, supersonic only, and both without the surrogate throwing an error.

### CODE REVIEW

File: trunk/SUAVE/Analyses/Aerodynamics/Vortex_Lattice.py [hunk 1]
```diff
@@ -245,14 +253,24 @@ def evaluate_surrogate(self,state,settings,geometry):
         
         for wing in geometry.wings.keys(): 
             inviscid_wing_lifts      = np.zeros([data_len,1])
```
Comment (rachealerhard) [inline trunk/SUAVE/Analyses/Aerodynamics/Vortex_Lattice.py:270]: Looks like this gets overwritten in the following if-else statement

File: trunk/SUAVE/Analyses/Aerodynamics/Vortex_Lattice.py [hunk 2]
```diff
@@ -245,14 +253,24 @@ def evaluate_surrogate(self,state,settings,geometry):
...
-            inviscid_wing_drags      = np.zeros([data_len,1])            
-            inviscid_wing_lifts = h_sub(Mach)*wing_CL_surrogates_sub[wing](AoA,Mach,grid=False)    + \
-                                    (h_sup(Mach) - h_sub(Mach))*wing_CL_surrogates_trans[wing]((AoA,Mach))+ \
-                                    (1- h_sup(Mach))*wing_CL_surrogates_sup[wing](AoA,Mach,grid=False)
+            inviscid_wing_drags      = np.zeros([data_len,1])
```
Comment (rachealerhard) [inline trunk/SUAVE/Analyses/Aerodynamics/Vortex_Lattice.py:271]: Also looks like this gets overwritten so I would remove this too

**Review by rachealerhard — APPROVED** [review]
I like the changes! Two small things to remove in the comments.

**Comment by planes** [comment]
Well, the appveyor failed for reasons that have nothing to do with SUAVE at all.

**Pull request merged**